README.rmd

```diff
@@ -0,0 +1,268 @@
+```
+
+## Quick start
+
+[![Documentation](https://img.shields.io/badge/Package-documentation-lightgrey.svg?style=flat)](https://epiforecasts.io/covidregionaldata/)
+
+### National data  
+
+To get worldwide time-series data by country (sourced from the WHO), use `get_national_data()`:
+``` r
+covidregionaldata::get_national_data()
+```
+
+```{r getalldata, message=F, results='hide'}
+allcountries <- covidregionaldata::get_national_data()
+```
+```{r showalldata, message=F, echo=F}
+allcountries <- allcountries %>% arrange(desc(date))
+kable(head(allcountries), caption = "All country Covid-19 data")
+```
+
+Using this data we can compare case information between countries, for example here is the number of deaths over time for each country in the G7:
+
+```{r plotg7, warning=F, message=F}
+g7 = c("United States", "United Kingdom", "France", "Germany", "Italy", "Canada", "Japan")
+allcountries %>%
+  filter(country %in% g7) %>% 
+  ggplot(aes(x=date, y=deaths_total, color=country))+
+  geom_line()+
+  labs(x=NULL, y="Reported Covid-19 deaths")+
+  theme_minimal()
+```
+
+
+### National regions
+
+To get time-series data for national regions of a specific country, for example by region in the UK, use `get_regional_data()`:
+
+```{r getukregions, message=F, results='hide'}
+ukdata <- covidregionaldata::get_regional_data(country = "UK")
+```
+```{r showukregions, message=F, echo=F}
+ukdata <- ukdata %>% arrange(desc(date))
+kable(head(ukdata), caption = "Uk regional Covid-19 data")
+```
+
+Now we have the data we can create plots, for example the evolution of the number of cases for each region:
+
+```{r plotukregions, warning=F, message=F}
+ukdata %>%
+  ggplot(aes(x=date, y=cases_total, color=region))+
+  geom_line()+
+  labs(x=NULL, y="Reported Covid-19 cases")+
+  theme_minimal()
+```
+
+### Subnational regions
+
+To get time-series data for subnational regions of a specific country, for example by local authority in the UK, use `include_level_2_regions = TRUE`:
+
+```{r getuksubregions, message=F, results='hide'}
+ukdata_sub <- covidregionaldata::get_regional_data(country = "UK", include_level_2_regions = T)
+```
+```{r showuksubregions, message=F, echo=F}
+ukdata_sub <- ukdata_sub %>% arrange(desc(date))
+kable(head(ukdata_sub), caption = "Uk sub-national regional Covid-19 data")
+```
+
+With the sub-national data we can evaluate the evolution of cases for local authorities within each region, for example the East Midlands:
+
+```{r plotuksubregions, warning=F, message=F}
+ukdata_sub %>%
+  filter(region %in% c("East Midlands")) %>% 
+  ggplot(aes(x=date, y=cases_total, color=authority))+
+  geom_line()+
+  labs(x=NULL, y="Reported Covid-19 cases")+
+  theme_minimal()
+```
+
+Subnational regions are only available in some countries. See below section 4, "Coverage".
+
+For further demonstrations see [Quick start vignette](https://github.com/epiforecasts/covidregionaldata/blob/master/vignettes/quickstart.Rmd#L84)
+
+## Usage
```

Comment:
I think we can get rid of all of this for now and put into a vignette in the future.